packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -1770,11 +1784,12 @@ module.exports = {
    * @returns {Boolean}
    */
   isPluginDocumentationNeeded: function(pluginName) {
-    const pluginRoutesWithDescription = this.getPluginRoutesWithDescription(
-      pluginName
-    );
-
-    return pluginRoutesWithDescription.length > 0;
+    const pluginsDocWantedByUser = strapi.plugins.documentation.config['x-strapi-config'].pluginsForWhichToGenerateDoc;
```

Comment:
can you destructure the config to the var `pluginsForWhichToGenerateDoc` instead of renaming it :)

Reply:
Done